app/src/components/Navbar/index.tsx

```diff
@@ -438,6 +444,21 @@ function Navbar(props: Props) {
                                 </DropdownMenuItem>
                             </TabList>
                             <div className={styles.optionBorder} />
+                            <TabPanel
+                                name="operational-learnings"
+                                className={styles.optionDetail}
```

Comment:
`optionDetail` selector is not defined. Let's define selectors in style files before  passing them as classNames.